client/src/components/Carousel.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import CarouselItem from './CarouselItem.jsx';
+
+class Carousel extends React.Component {
+  constructor() {
```

Comment:
Is there no need to pass in props?

Reply:
Hey Brian,
There will definitely be a need for props as I flesh out this component, but at this stage it's just a skeleton. I'll add props when I'm at the stage where I'm using data.